src/Entity/OgMembership.php

```diff
@@ -74,78 +80,75 @@ public function setCreatedTime($timestamp) {
   }
 
   /**
-   * @param mixed $etid
-   *
-   * @return OgMembership
+   * {@inheritdoc}
    */
   public function setUser($etid) {
     $this->set('uid', $etid);
     return $this;
   }
 
   /**
-   * @param mixed $fieldName
-   *
-   * @return OgMembership.
+   * {@inheritdoc}
+   */
+  public function getUser() {
+    return $this->get('uid')->value;
+  }
```

Comment:
Does this get the User object or the ID? The documentation on the interface says it returns the object, but I'm not sure if this will be correct.

For example, it is allowed to pass an ID to `::setUser()`, and from looking at the code it seems like this will return an ID instead of a User object:

```
$membership = OgMembership::create();
$membership->setUser(1);
$user = $membership->getUser();
```

This is not strictly related to the scope of this issue, which is only adding the interface, so I'm fine to investigate this in a follow-up issue.